src/main/java/moze_intel/projecte/utils/PlayerHelper.java

```diff
@@ -134,7 +134,7 @@ public static BlockPos getBlockLookingAt(EntityPlayer player, double maxDistance
 	public static boolean hasBreakPermission(EntityPlayerMP player, BlockPos pos)
 	{
 		return hasEditPermission(player, pos)
-				&& ForgeHooks.onBlockBreakEvent(player.getEntityWorld(), player.interactionManager.getGameType(), player, pos) != -1;
+				&& !(ForgeHooks.onBlockBreakEvent(player.getEntityWorld(), player.interactionManager.getGameType(), player, pos) == -1);
```

Comment:
explain?

Reply:
Can not remember why, but this works better.

Remove? Hm.. It changes something? No. Why do not just use this version?

Reply:
Because you cannot provide an explanation for why to use your version or why your version is better? Also this line probably doesn't change a single thing....

`!(a == -1)` <=> `(a != -1)` ...